File: Python/tests/test_igt_pvl_delta.py
import pytest

from hbayesdm.models import igt_pvl_delta


def test_igt_pvl_delta():
    _ = igt_pvl_delta(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_pvl_delta.R
context("Test igt_pvl_delta")
library(hBayesDM)

test_that("Test igt_pvl_delta", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_pvl_delta(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
